src/couch_replicator_scheduler.erl

```diff
@@ -337,7 +355,11 @@ stop_job_int(#job{} = Job, State) ->
 
 -spec remove_job_int(#job{}) -> true.
 remove_job_int(#job{} = Job) ->
-    ets:delete(?MODULE, Job#job.id).
+    DeleteRes = ets:delete(?MODULE, Job#job.id),
+    TotalJobs = ets:info(?MODULE, size),
+    couch_stats:update_gauge([couch_replicator, jobs, total], TotalJobs),
+    couch_stats:increment_counter([couch_replicator, jobs, removes]),
```

Comment:
we increment remove even if we didn't remove anything. same solution as for add_job, move this to the {ok, Job} clause of add_job